web/app/scripts/nominatim/module.js

```diff
@@ -0,0 +1,8 @@
+(function () {
+    'use strict';
+
+    angular.module('driver.nominatim', [
```

Comment:
Probably an annoying request, but I think it would be nicer if all of these references to `nominatim` (module, service, variables) were changed to something more general and self-explanatory, like `geocoder`. It would also make it less confusing if we were to do something such as switching geocoders in the future.